src/browser/components/molecules/Task/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react'
+import Layout from '../../molecules/Layout/index'
+import styles from './index.css'
+const moment = require('moment')
+
+
+
+export default class Task extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      complete: false
+    }
+    this.isCompleteHandler = this.isCompleteHandler.bind(this)
```

Comment:
I think we do need to bind this.
If I take this line out, the button doesn't work and I get the error "Cannot read property 'props' of null at isCompleteHandler "